internal/executor/terminalwrapper/terminalwrapper.go

```diff
@@ -117,6 +143,21 @@ func (wrapper *Wrapper) Wait() chan Operation {
 				return
 			}
 
+			if !wrapper.lifecycleExpiringSent {
```

Comment:
Is it possible to have two events? For example, you get the first one, open terminal, do some stuff and go away again. 🤔

Reply:
Implemented in https://github.com/cirruslabs/cirrus-ci-agent/pull/192/commits/a43e2f445bd6329927c5b4919c961147b0e43523.

The terminal is now waited in `CIRRUS_TERMINAL_EXPIRATION_WINDOW`-second windows.